src/modules/media/types/index.ts

```diff
@@ -5,7 +5,6 @@ export interface Media {
 	schemaIdentifier: string; // Unique id per object
 	meemooIdentifier: string; // PID (DON'T use this for identification of an object)
 	premisIdentifier: Record<string, string[]>;
-	premisRelationship: Record<string, string[]>;
 	premisIsPartOf?: string;
```

Comment:
moeten we niet het veld premisIsPartOf wegnemen ipv premisRelationship?

Volgens de metadata set op confluence:
In de taak zeggen ze: alles onder relations wegnemen:
![image](https://user-images.githubusercontent.com/1710840/193274729-233e976a-95de-49fb-bf99-dee1a27c1943.png)